server/controllers/shared/risksAndNeedsController.ts

```diff
@@ -25,6 +26,28 @@ export default class RisksAndNeedsController {
     private readonly referralService: ReferralService,
   ) {}
 
+  attitudes(): TypedRequestHandler<Request, Response> {
+    return async (req: Request, res: Response) => {
+      TypeUtils.assertHasUser(req)
+
+      const sharedPageData = await this.sharedPageData(req, res)
+
+      const attitudes = await this.oasysService.getAttitude(req.user.username, sharedPageData.referral.prisonNumber)
```

Comment:
plural = getSingular. Perhaps the variable should be singular here since it's holding a singular instance of the singular-named entity?